src/homeWork/module5/task2/DisplayInfoFromPerson.java

```diff
@@ -8,15 +10,29 @@
 
 public class DisplayInfoFromPerson {
     public static void main(String[] args) {
-        Person firstPerson = new Person("FirstPersonName", "FirstPersonSurname", "FirstPersonEmail");
-        Person secondPerson = new Person();
+        HashMap<Passport, Person> hashMap = new HashMap<Passport, Person>();
```

Comment:
Code convention: use interface rather then implementation as Type.
Variable name should be meaningful, `hashMap` non compliant name, `personInfo` compliant
`Map<Passport, Person> personInfo= new HashMap<Passport, Person>();`